feflow/settings/neq_cycling.py

```diff
@@ -44,12 +46,18 @@ class NonEquilibriumCyclingSettings(Settings):
     class Config:
         arbitrary_types_allowed = True
 
+    # System Settings (from openfe)
+    system_settings: SystemSettings
+    forcefield_cache: Optional[str] = "db.json"  # TODO: Remove once it has been integrated with openfe settings
+
+    # Solvation settings
+    solvation_settings: SolvationSettings
+
     # Lambda settings
     lambda_functions = DEFAULT_ALCHEMICAL_FUNCTIONS
 
     # alchemical settings
-    softcore_LJ_v2 = True
-    interpolate_old_and_new_14s = False
+    alchemical_settings: AlchemicalSettings
```

Comment:
There may be some settings, such as `lambda_windows`, which don't actually apply here, I think?

Reply:
Yes, definitely, there are settings in `AlchemicalSettings` that do not apply to this protocol,  but I think we would need to stick with these for now until we can come up with a base alchemical settings object that should cover the common settings for both protocols. Raised the issue in https://github.com/choderalab/feflow/issues/6